python/nano/src/bigdl/nano/deps/onnxruntime/tensorflow/tensorflow_onnxruntime_model.py

```diff
@@ -21,7 +21,8 @@
 from tempfile import TemporaryDirectory
 import tensorflow as tf
 from bigdl.nano.utils.common import get_default_args
-from bigdl.nano.utils.tf import KERAS_VERSION_LESS_2_10, fake_tensor_from_spec
+from bigdl.nano.utils.tf import KERAS_VERSION_LESS_2_10
+from bigdl.nano.utils.tf import tensor_spec_to_shape
```

Comment:
many import are changed to this format (1 import for 1 line). I wonder are we following any coding style for this?

Reply:
I think it's just personal preference